### PR TITLE
Ensure the VKB locale matches the chosen language

### PR DESCRIPTION
**Pull request by chriadam**
Contributes to issue #1632

### CODE REVIEW

File: components/InputPanel.qml [hunk 1]
```diff
@@ -128,8 +130,9 @@ QtVirtualKeyboard.InputPanel {
 		}
 	}
 
-
+	onLocaleNameChanged: VirtualKeyboardSettings.locale = localeName
 	Component.onCompleted: {
 		VirtualKeyboardSettings.activeLocales = ["en_US", "cs_CZ", "da_DK", "de_DE", "es_ES", "fr_FR", "it_IT", "nl_NL", "pl_PL", "ru_RU", "ro_RO", "sv_SE", "th_TH", "tr_TR", "uk_UA", "zh_CN", "ar_AR"]
+		VirtualKeyboardSettings.locale = localeName
```
Comment (DanielMcInnes) [inline components/InputPanel.qml:136]: If you change the gui language to Arabic, we change Language.currentLocaleName to 'ar_EG' (Arabic - Egypt), which is not supported by the vkb, so the changes don't get applied. The only Arabic that the vkb knows about is 'ar_AR' The vkb will stay on the previously selected language. Apart from that, lgtm.

Comment (DanielMcInnes) [reply components/InputPanel.qml:136]: FYI, vkb supports these locales: 
[ar_AR,bg_BG,cs_CZ,da_DK,de_DE,el_GR,en_GB,en_US,es_ES,es_MX,et_EE,fa_FA,fi_FI,fr_CA,fr_FR,he_IL,hi_IN,hr_HR,hu_HU,id_ID,it_IT,ja_JP,ko_KR,ms_MY,nb_NO,nl_NL,pl_PL,pt_BR,pt_PT,ro_RO,ru_RU,sk_SK,sl_SI,sq_AL,sr_SP,sv_SE,th_TH,tr_TR,uk_UA,vi_VN,zh_CN,zh_TW]

Comment (chriadam) [reply components/InputPanel.qml:136]: Thanks!  I will add a hardcoded fixup for this case.

**Review by DanielMcInnes — APPROVED** [review]
lgtm

**Pull request merged**